schema/logging_backends.hcl

```diff
@@ -0,0 +1,48 @@
+enum "source" {
+  schema = schema.public
+  values = ["KubernetesCRD", "ConfigFile"]
+}
+
+table "logging_backends" {
+  schema = schema.public
+  column "id" {
+    null    = false
+    type    = uuid
+    default = sql("generate_ulid()")
+  }
+  column "name" {
+    null = false
+    type = text
+  }
+  column "labels" {
+    null = true
+    type = jsonb
+  }
+  column "spec" {
+    null = false
+    type = jsonb
+  }
+  column "source" {
+    null = true
+    type = enum.source
+  }
+  column "created_at" {
+    null = true
+    type = timestamptz
```

Comment:
I think this should be timestamp, when we switch to timestampz it should be for everything

Reply:
I was thinking the reverse, that lets use timestamptz for new tables, and do a big bang for everything later